libraries/botbuilder-dialogs-adaptive/src/actions/telemetryTrackEventAction.ts

```diff
@@ -6,14 +6,18 @@
  * Licensed under the MIT License.
  */
 
-import { StringExpression, BoolExpression } from 'adaptive-expressions';
-import { Dialog, DialogContext, DialogTurnResult } from 'botbuilder-dialogs';
-import { Converter } from 'botbuilder-dialogs-declarative';
+import {
+    StringExpression,
+    BoolExpression,
+    BoolExpressionConverter,
+    StringExpressionConverter,
+} from 'adaptive-expressions';
+import { Converter, Converters, Dialog, DialogContext, DialogTurnResult } from 'botbuilder-dialogs';
 
 /**
  * Converter to convert telemetry properties configuration.
  */
-export class TelemetryPropertiesConverter implements Converter {
+class TelemetryPropertiesConverter implements Converter<object, { [name: string]: StringExpression }> {
```

Comment:
I think the linter has been complaining about using `object`. `Record<string, unknown>` should work better.